src/NodaTime.Web.Test/Helpers/VersionComparerTest.cs

```diff
@@ -0,0 +1,23 @@
+﻿using NodaTime.Web.Helpers;
+using NUnit.Framework;
+
+namespace NodaTime.Web.Test.Helpers
+{
+    public class VersionComparerTest
+    {
+        [Test]
+        [TestCase("1.0.0", "1.0.1")]
+        [TestCase("1.0.0", "1.1.0")]
+        [TestCase("1.1.0", "2.0.0")]
+        [TestCase("1.0.0", "1.0.1-alpha01")]
+        [TestCase("1.0.0-alpha01", "1.0.0-alpha02")]
+        [TestCase("1.0.0-alpha01", "1.0.0-beta01")]
+        [TestCase("1.0.0-alpha01", "1.0.0")]
+        public void Compare(string earlier, string later)
```

Comment:
Good plan, will add an extra test for equivalent versions.

Will actually fix it for two-digit major/minor/patch, otherwise it'll bug me :)